src/main/java/org/ssu/edu/teachua/ui/components/header/HeaderComponent.java

```diff
@@ -40,7 +40,7 @@ public class HeaderComponent extends BaseComponent {
     private WebElement locationButton;
 
     @FindBy(how = How.XPATH, using =
-            "//ul[contains(@class, 'dropdown-menu-vertical')]//li[contains(@class, 'child')]"
+            "(//ul[contains(@class, 'dropdown-menu-vertical')])[2]//li[contains(@class, 'child')]"
```

Comment:
$x( "(//div[contains(@class, 'ant-dropdown-show-arrow') and not(contains(@class, 'ant-dropdown-hidden'))])//ul//li")